webpush/encrypt.go

```diff
@@ -139,36 +142,64 @@ type EncryptionResult struct {
 	ServerPublicKey []byte
 }
 
+// Decrypt an encrypted messages.
```

Comment:
If we have decrypt then it should be in here. Perhaps pull it into a separate PR so that we can discuss it separately?